unitTests/ui/MLFacet/MLFacetTest.ts

```diff
@@ -143,30 +149,31 @@ export function MLFacetTest() {
     });
 
     it('allows to showMoreValues, uses numberOfValues by default as the amount', () => {
-      spyOn(test.cmp.mLFacetQueryController, 'increaseNumberOfValuesToRequest');
-
+      const additionalNumberOfValues = test.cmp.options.numberOfValues;
       test.cmp.showMoreValues();
-
-      expect(test.cmp.mLFacetQueryController.increaseNumberOfValuesToRequest).toHaveBeenCalledWith(test.cmp.options.numberOfValues);
       expect(test.cmp.queryController.executeQuery).toHaveBeenCalled();
+
+      simulateRequest();
+      expect(facetRequest.numberOfValues).toBe(test.cmp.options.numberOfValues + additionalNumberOfValues);
     });
 
     it('allows to showMoreValues with a custom amount of values', () => {
-      spyOn(test.cmp.mLFacetQueryController, 'increaseNumberOfValuesToRequest');
-
-      test.cmp.showMoreValues(56);
-
-      expect(test.cmp.mLFacetQueryController.increaseNumberOfValuesToRequest).toHaveBeenCalledWith(56);
+      const additionalNumberOfValues = 38;
+      test.cmp.showMoreValues(additionalNumberOfValues);
       expect(test.cmp.queryController.executeQuery).toHaveBeenCalled();
```

Comment:
It would be better to break this test into two; one for the executeQuery call, one for the number of values sent in the facet request. When the test fails, the test description will better explain the error.